packages/core-mobile/app/screens/bridge/hooks/useBridge.ts

```diff
@@ -1,268 +1,214 @@
-} from '@avalabs/core-bridge-sdk'
 import { useEffect, useMemo, useState } from 'react'
-import { VsCurrencyType } from '@avalabs/core-coingecko-sdk'
-import { useBtcBridge } from 'screens/bridge/hooks/useBtcBridge'
-import { useEthBridge } from 'screens/bridge/hooks/useEthBridge'
-import { useAvalancheBridge } from 'screens/bridge/hooks/useAvalancheBridge'
-import { AssetBalance, BridgeProvider } from 'screens/bridge/utils/types'
+import { AssetBalance } from 'screens/bridge/utils/types'
 import Big from 'big.js'
-import { useSelector } from 'react-redux'
-import { selectSelectedCurrency } from 'store/settings/currency'
-import { selectActiveAccount } from 'store/account'
-import { BIG_ZERO, bigToBigInt } from '@avalabs/core-utils-sdk'
 import Logger from 'utils/Logger'
-import BridgeService from 'services/bridge/BridgeService'
-import { selectIsDeveloperMode } from 'store/settings/advanced'
-import { selectBridgeAppConfig } from 'store/bridge'
-import UnifiedBridgeService from 'services/bridge/UnifiedBridgeService'
-import { useNetworks } from 'hooks/networks/useNetworks'
+import { Network } from '@avalabs/core-chains-sdk'
+import { BridgeAsset, BridgeType } from '@avalabs/bridge-unified'
 import { useNetworkFee } from 'hooks/useNetworkFee'
-import { bigintToBig } from 'utils/bigNumbers/bigintToBig'
-import { useCoinGeckoId } from 'hooks/useCoinGeckoId'
-import { useSimplePrice } from 'hooks/useSimplePrice'
-import { isUnifiedBridgeAsset, networkToBlockchain } from '../utils/bridgeUtils'
-import { useUnifiedBridge } from './useUnifiedBridge/useUnifiedBridge'
-import { getTargetChainId } from './useUnifiedBridge/utils'
+import { getAssetBalance, unwrapAssetSymbol } from '../utils/bridgeUtils'
+import { useAssetBalances } from './useAssetBalances'
+import { useBridgeAssets } from './useBridgeAssets'
+import {
+  useGetBridgeFees,
+  useGetMinimumTransferAmount
+} from './useGetBridgeFees'
+import { useBridgeAssetPrice } from './useBridgeAssetPrice'
+import { useBridgeType } from './useBridgeType'
+import { useBridgeTransfer } from './useBridgeTransfer'
+import {
+  useBridgeSourceNetworks,
+  useBridgeTargetNetworks
+} from './useBridgeNetworks'
 
-export interface BridgeAdapter {
-  address?: string
-  sourceBalance?: AssetBalance
-  targetBalance?: AssetBalance
+interface Bridge {
+  assetBalance?: AssetBalance
   assetsWithBalances?: AssetBalance[]
-  networkFee?: Big
+  networkFee?: bigint
   loading?: boolean
-  bridgeFee?: Big
-  /** Amount minus network and bridge fees */
-  receiveAmount?: Big
+  bridgeFee: bigint
   /** Maximum transfer amount */
-  maximum?: Big
+  maximum?: bigint
   /** Minimum transfer amount */
-  minimum?: Big
-  wrapStatus?: WrapStatus
+  minimum?: bigint
   /**
-   * Transfer funds to the target blockchain
+   * Transfer funds to the target network
    * @returns the transaction hash
    */
-  transfer: () => Promise<string | undefined>
+  transfer: (bridgeType: BridgeType) => Promise<string | undefined>
```

Comment:
it looks like bridgeType is already available inside this hook. no need to pass as param?

Reply:
nice catch! fixed.